xwiki-platform-core/xwiki-platform-security/xwiki-platform-security-authorization/xwiki-platform-security-authorization-api/src/main/java/org/xwiki/security/authorization/cache/internal/DefaultSecurityCache.java

```diff
@@ -664,10 +655,30 @@ private SecurityCacheEntry getEntry(UserSecurityReference userReference, Securit
      *         cache.
      */
     private SecurityCacheEntry getShadowEntry(SecurityReference userReference, SecurityReference wiki)
+    {
+        return getInternal(getShadowEntryKey(userReference, wiki));
+    }
+
+    private SecurityCacheEntry getInternal(String key)
```

Comment:
Could be interesting to have an explanation of why getInternal is required instead of a simple get.